app/src/app/components/sidebar/ResetMapButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { useMapStore } from "@/app/store/mapStore";
+
+export function ResetMapButton() {
+  const mapStore = useMapStore.getState();
+
+  const handleClickResetMap = () => {
+    mapStore.setFreshMap(true);
+  };
+
+  return (
+    <button
+      onClick={handleClickResetMap}
+      className=" m-3 p-1.5 bg-white rounded-lg shadow-md"
+    >
```

Comment:
I: Use radix themes [Button](https://www.radix-ui.com/themes/docs/components/button) for consistency + accessibility.